test/functional/apps/getting_started/_shakespeare.js

```diff
@@ -88,6 +88,7 @@ export default function ({ getService, getPageObjects }) {
     */
     it('should configure Terms aggregation on play_name', async function () {
       await PageObjects.visualize.clickBucket('X-Axis');
+      await PageObjects.common.sleep(1000);
```

Comment:
There's already a 500ms sleep in the clickBucket method. 
https://github.com/liza-mae/kibana/blob/3e3f76d09b6eb916249c1c833b2c7d3613dfa0db/test/functional/page_objects/visualize_page.js#L397

Should we just increase that sleep instead of adding this one to the test?

Reply:
Unfortunately what I found is sometimes specific tests take longer and that was the case here, these were run in a loop and adding the sleep there helped, but I see some changes are pending for clickBucket, so let me take those in.